src/state/routing/slice.ts

```diff
@@ -85,7 +93,7 @@ export const routingApi = createApi({
     baseUrl: 'https://api.uniswap.org/v1/',
   }),
   endpoints: (build) => ({
-    getQuote: build.query<GetQuoteResult, GetQuoteArgs>({
+    getQuote: build.query<TradeResult, GetQuoteArgs>({
```

Comment:
Why no `SkipToken`? I think (from widgets code) that you may also need to check for the skip token and return early, but I am not sure. See https://github.com/Uniswap/widgets/blob/main/src/state/routing/slice.ts#L31.

```suggestion
    getQuote: build.query<TradeResult, GetQuoteArgs | SkipToken>({
```

Reply:
you don't need to explicitly handle that - it's baked into rtk-query!